### ISSUE TITLE
How to import existing GPG keypair

### ISSUE THREAD

**Issue by Eggo-Plant** [mailpile/Mailpile#2288]
I'm having trouble figuring out what directory that Mailpile reads gpg keypairs from. Is it possible to import my own gpg keys?

**Comment by h3artbl33d** [comment]
Yes, you can import your own GPG key. There are two routes on how Mailpile handles GPG keys:

* Using the system GnuPG; I believe this is the default.
* Using an internal library, which can be set through the Mailpile settings.

You want to use a keypair corresponding to a mail account you have setup in Mailpile?

**Comment by Eggo-Plant** [comment]
What I'm trying to do is import my own gpg key that I generated outside of mailpile.

**Comment by edo0** [comment]
@Eggo-Plant  You need to import it in you system GnuPG, from which Mailpile will take it up automatically
[https://unix.stackexchange.com/questions/184947/how-to-import-secret-gpg-key-copied-from-one-machine-to-another](https://unix.stackexchange.com/questions/184947/how-to-import-secret-gpg-key-copied-from-one-machine-to-another)
